app/boards/shields/Kconfig.defconfig

```diff
@@ -0,0 +1,14 @@
+
+
+
+config ZMK_KEYBOARD_NAME
+	default "cradios"
+
+# Unable to use interrupts as the same pin number is used
+# across A & B controllers, and STM32F303CCT6 can't enable
+# interrutps for multiple controllers for the same "line"
+# for the external interrupts.
+config ZMK_KSCAN_GPIO_POLLING
```

Comment:
Were you testing with proton-c controllers? If so, there's actually a way to provide board specific overrides for a given shield, you can see that in https://docs.zephyrproject.org/2.3.0/guides/porting/shields.html?highlight=shield#board-specific-shield-configuration

I would remove this from the "base" defconfig here, and put it in the proton-c specific one.

Reply:
I'm not clear what you mean here so i just deleted everything about config...